src/lumigo_tracer/wrappers/http/sync_http_wrappers.py

```diff
@@ -29,10 +30,19 @@
 HookedData = namedtuple("HookedData", ["headers", "path"])
 
 
+def is_lumigo_edge(host: Optional[str]):
```

Comment:
This solves the second bug: when we catch our own spans.
We just need to ignore them.